multicluster-config/observer-values.yaml

```diff
@@ -0,0 +1,144 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Values for dNation Kubernetes Monitoring Stack to be used in addition
+# to default values.yaml for deploying thanos components for multicluster
+# monitorig at observer
+# Declare variables to be passed into helm chart dependencies.
+
+thanosStorage:
+  secret:
+    name: thanos-objstore-config
+    key: objstore.yml
+  # More details about supported storage types and config options can be found here: https://thanos.io/tip/thanos/storage.md/
+  config: ""
+
+thanosQueryEnvoySidecar:
+  enabled: true
+  # define whole envoy-config as string
+  #  envoyConfigOverride: ""
+  #  envoyConfig:
+  service:
+    name: thanos-query-envoy
+    labels:
+      name: thanos-query-envoy
+    selector:
+      app.kubernetes.io/component: query
+      app.kubernetes.io/name: thanos
+  config: []
+
+## Configure additional grafana datasources
+grafanaDatasourcesAsConfigMap:
+  cluster-metrics:
+    - name: thanos
+      isDefault: true
+      type: prometheus
+      access: proxy
+      url: "http://{{ .Release.Name }}-thanos-query-frontend:9090"
+thanos:
+  enabled: true
+  existingObjstoreSecret: thanos-objstore-config
+  queryFrontend:
+    enabled: true
+  query:
+    replicaLabel:
+      - prometheus_replica
+    dnsDiscovery:
+      sidecarsService: kube-prometheus-thanos-discovery
+      sidecarsNamespace: "{{ .Release.Namespace }}"
+    # URLs of exposed Thanos query components on remote workload clusters
+    stores: []
+
+    # Envoy sidecar for communication with workload clusters
+    sidecars:
+    - name: envoy-sidecar
+      image: 'envoyproxy/envoy:v1.20.1'
+      imagePullPolicy: IfNotPresent
+      args:
+        - '-c'
+        - /config/envoy.yaml
+      ports:
+        - name: egress-http-1
+          containerPort: 10001
+          protocol: TCP
+      resources: {}
+      volumeMounts:
+        - name: envoy-config
+          mountPath: /config
+          mountPropagation: None
+        - name: envoy-certs
+          mountPath: /certs
+          mountPropagation: None
+      terminationMessagePath: /dev/termination-log
+      terminationMessagePolicy: File
+
+    extraVolumes:
+      - name: envoy-config
+        configMap:
+          name: thanos-query-envoy-config
+          defaultMode: 420
+          optional: false
+      - name: envoy-certs
+        secret:
+          secretName: thanos-query-envoy-certs
+          defaultMode: 420
+          optional: false
+
+  bucketweb:
+    enabled: true
+  compactor:
+    enabled: true
+  storegateway:
+    enabled: true
+  ruler:
+    enabled: false
+
+kube-prometheus-stack:
+  enabled: true
+#  # disable because workloads prometheuses are not conencted to alertmanagers
```

Comment:
What is this comment?

Reply:
This is not true anymore, so can be deleted